nasa/harmony-regression-tests
# Ensure Harmony regression test suite runs tests.

## Description

This PR arises from something @flamingbear spotted during DAS-1745. The `pytest` invocation was behaving weirdly and not actually running the code to test which services are deployed.

In this PR, I've ripped out `pytest`, as `papermill` will catch any failed assertions in a Jupyter notebook cell as a failed test. I've also removed the `util.py` module, as much of it was unused, and it seemed like a little overhead for the single test we have.

## Jira Issue ID

N/A (discovered via DAS-1745)

## Local Test Steps

* Pull this branch.
* In the `test` directory run `make harmony-image`
* Ensure the requisite environment variables are set (`HARMONY_HOST_URL`, `EDL_USER`, `EDL_PASSWORD`.
* In the `test` directory run `./run_notebooks.sh harmony`. It should complete with success.
* Fire up a Jupyter notebook server and look at `test/output/harmony/Results.ipynb` to ensure the correct version of the notebook ran.
* If you want to ensure the test actually is doing anything, change one of the `expected_service_names` to something random and repeat the steps above - the test should fail.

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
